typescript/sdk/test/gas/calculator.test.ts

```diff
@@ -4,14 +4,14 @@ import sinon from 'sinon';
 
 import { utils } from '@abacus-network/utils';
 
+import { Chains } from '../../src/consts/chains';
+import { AbacusCore } from '../../src/core/AbacusCore';
+import { CoreContracts } from '../../src/core/contracts';
 import {
-  AbacusCore,
-  Chains,
   InterchainGasCalculator,
-  MultiProvider,
-} from '../..';
-import { CoreContracts } from '../../src';
-import { ParsedMessage } from '../../src/gas/calculator';
+  ParsedMessage,
+} from '../../src/gas/calculator';
+import { MultiProvider } from '../../src/providers/MultiProvider';
```

Comment:
how do you feel about separate test dirs versus test.ts embedded in src?

Reply:
I prefer embedded in src, you?

Reply:
I personally also prefer co-location

Reply:
Cool I'll move these and colocate them then